docs/content/policy-language.md

```diff
@@ -3772,79 +3661,17 @@ For a reference on JSON Schema please see: <http://json-schema.org/understanding
-`input` and `data` reserved keywords | `input` and `data` are reserved keywords, and may not be used as names for rules and variable assignment.                                                                                                                                                      | 1.0
-Use of deprecated built-ins | Use of deprecated functions is prohibited, and these will be removed in OPA 1.0. Deprecated built-in functions: `any`, `all`, `re_match`,  `net.cidr_overlap`, `set_diff`, `cast_array`, `cast_set`, `cast_string`, `cast_boolean`, `cast_null`, `cast_object` | 1.0
-
-{{< info >}}
-If the `rego.v1` import is present in a module, all strict mode checks documented above except the unused local assignment and unused imports checks are enforced on the module.
-
-Additionally the `rego.v1` import also requires the usage of `if` and `contains` keywords when declaring certain rules. The `if` keyword is required before a rule body and the `contains` keyword is required for partial set rules.
-{{< /info >}}
-
-## The `rego.v1` Import
-
-In the future, when [OPA v1.0](../opa-1) is released, breaking changes will be introduced to the Rego language.
-The `rego.v1` import is a way to opt-in to these breaking changes early, and ensure that your policies are compatible with OPA v1.0.
-If a module containing this import is not compatible with OPA v1.0, it will cause a compilation error.
-
-When a module imports `rego.v1`, the following features and constraints are implied:
-
-* all [Future keywords](#future-keywords) are implied and can be used without import.
-  These imports are mutually exclusive, and it will cause a compilation error to import both `rego.v1` and `future.keywords` in the same module.
-* the [if](#futurekeywordsif) keyword is required before a rule body declaration.
-* the [contains](#futurekeywordscontains) keyword is required for partial set (multi-value) rules.
-* most [Strict mode](#strict-mode) constraints and checks are implied and enforced. See the strict mode [constraints and checks table](#strict-mode-constraints-and-checks) for details.
-
-The `rego.v1` import only affects the module where it's declared. It does not affect any other modules, even if they are importing, or is imported by, a module where `rego.v1` is declared.
-
-In OPA v1.0, the `rego.v1` import will have no semantic impact on the policy, as all its implied features and constraints will be enforced by default. It will however still be a valid statement, and won't cause any compilation errors.
-
-Example policy that imports `rego.v1` to be compatible with the future syntax in OPA v1.0:
-
-```live:rego_v1:module:read_only
-package example
+## Compilation Constraints and Checks
 
-import rego.v1
+Below constraints and safety checks are enforced by default during compilation:
```

Comment:
The compiler has dozens more constraints and checks, some far more vital than these (recursion, type checks, etc.). These are only special because they were part of strict-mode in v0.x, but are now part of the default set. I think we can make note of this difference in the v0 docs, but the below table fells out of place; unless we add all other constraints and checks too, which I think is overkill.

Reply:
Makes sense. Moved this section to the v0 docs.